paddle/fluid/framework/new_executor/pir_adaptor/pir_adaptor_util.cc

```diff
@@ -280,6 +290,52 @@ void CheckInputVars(pir::Operation* op,
   }
 }
 
+void DeepCopyVariable(const Variable* src_var,
+                      Variable* dst_var,
+                      ValueExecutionInfo* value_exe_info,
+                      uint32_t stack_size) {
+  if (src_var->IsType<phi::DenseTensor>()) {
+    auto* tmp_grad_tensor = dst_var->GetMutable<phi::DenseTensor>();
```

Comment:
```suggestion
    auto* tmp_dst_tensor = dst_var->GetMutable<phi::DenseTensor>();
```
这里不需要耦合grad的概念，直接tmp_dst_tensor就可以？

Reply:
是的，不需要

Reply:
下个pr修改